mods/ra/rules/structures.yaml

```diff
@@ -520,6 +520,9 @@ DOME:
 	InfiltrateForExploration:
 	DetectCloaked:
 		Range: 10c0
+		UpgradeTypes: powered
+		UpgradeMaxEnabledLevel: 0
+		UpgradeMinEnabledLevel: -1
```

Comment:
`Power` and `CanPowerDown` both grant 1 level while "active" (for `CanPowerDown` this means when it *isn't* powered down), then `DetectCloaked` sets a `MinEnabledLevel` of 2.